packages/tooltip/src/components/tooltip/Tooltip.tsx

```diff
@@ -1,14 +1,84 @@
-import { Text } from "@stenajs-webui/core";
+import { Indent, Row, Text } from "@stenajs-webui/core";
+import {
+  Icon,
+  stenaExclamationCircle,
+  stenaExclamationTriangle,
+  stenaInfoCircle,
+} from "@stenajs-webui/elements";
+import TippyComponent, {
+  TippyProps as TippyComponentProps,
+} from "@tippyjs/react";
+import cx from "classnames";
 import * as React from "react";
-import { Popover, PopoverProps, PopoverVariant } from "../popover/Popover";
+import { TippyCallbackRef } from "../../hooks/UseTippyInstance";
+import styles from "./Tooltip.module.css";
 
-export interface TooltipProps extends Partial<Omit<PopoverProps, "content">> {
-  variant?: PopoverVariant;
+type TooltipVariant = "info" | "warning" | "error";
+
+interface TippyProps
+  extends Partial<Omit<TippyComponentProps, "theme" | "render">> {
+  tippyRef?: TippyCallbackRef<HTMLDivElement>;
+}
+
+export interface TooltipProps extends TippyProps {
+  variant?: TooltipVariant;
   label: string;
 }
 
-export const Tooltip: React.FC<TooltipProps> = ({ label, ...popoverProps }) => {
+const variantIcons = {
+  info: stenaInfoCircle,
+  warning: stenaExclamationCircle,
+  error: stenaExclamationTriangle,
+};
+
+export const Tooltip: React.FC<TooltipProps> = ({
+  visible,
+  trigger = "mouseenter",
+  children,
+  tippyRef,
+  delay = 0,
+  variant,
+  content,
+  maxWidth = "125px",
```

Comment:
Why the default maxWidth? This make the difference between single and multiple rows depend on the length of the text only.

This first story used to be one line and now it's two:
<img width="221" alt="image" src="https://user-images.githubusercontent.com/6987939/207045022-3e46e561-ec9e-4414-923c-28c35d959e3f.png">

Reply:
Do you suggest we use a boolean to tell it to be a single or multi row instead?